### ISSUE TITLE
Hide top on upscrolling

### ISSUE THREAD

**Issue by mariobehling** [fossasia/apps.loklak.org#77]
There is an effect on the web version of the Google app store (https://play.google.com): When the user scrolls up the top bar with the search box gets hidden and the following bar becomes the top bar. Please implement the same on apps.loklak.org. Hide the top bar on scroll up.

**Comment by Abhi2424shek** [comment]
Do you mean to the display of the search box on scroll up and hide it in scroll down?

**Comment by mariobehling** [comment]
@Abhi2424shek Hide the top bar same as https://play.google.com.

**Comment by Marauderer97** [comment]
@Abhi2424shek updates please?

**Comment by Abhi2424shek** [comment]
Well sorry I did not see that I was assigned, would give a PR @Marauderer97

**Comment by Abhi2424shek** [comment]
Well @mariibehling pls test it again. I think someone fixed it.

**Comment by jig08** [comment]
@Abhi2424shek Nope, it still needs work. Un-assigning you for the moment, let know if you still want to work on this...

**Comment by KukkadapuSushma** [comment]
Is someone working on this? OR can i take up this issue?

**Comment by jig08** [comment]
@KukkadapuSushma Go ahead !

**Comment by KukkadapuSushma** [comment]
Done, sending PR

**Comment by jig08** [comment]
@KukkadapuSushma Thanks, pls see to the failing tests...
